platform/qt/src/sqlite3.cpp

```diff
@@ -60,20 +61,29 @@ void checkDatabaseOpenError(const QSqlDatabase &db) {
     }
 }
 
+namespace {
+    static QAtomicInt count = 0;
```

Comment:
I would also move this to inside `incrementCount` so it cannot be accessed outside the function scope.